Flaky test `test_four_word_command`

<!-- (At least include the following, feel free to add if you have more content) -->

### RaftKeeper version
v2.0.4
### Description
2024-04-09T16:31:55.0083814Z ________________________________ test_cmd_crst _________________________________
2024-04-09T16:31:55.0084749Z 
2024-04-09T16:31:55.0085284Z started_cluster = <helpers.cluster_service.RaftKeeperCluster object at 0x7fd39adbfc40>
2024-04-09T16:31:55.0086213Z 
2024-04-09T16:31:55.0086410Z     def test_cmd_crst(started_cluster):
2024-04-09T16:31:55.0088449Z         zk = None
2024-04-09T16:31:55.0088867Z         try:
2024-04-09T16:31:55.0089211Z             wait_nodes()
2024-04-09T16:31:55.0089631Z             clear_znodes()
2024-04-09T16:31:55.0090065Z             reset_conn_stats()
2024-04-09T16:31:55.0090335Z     
2024-04-09T16:31:55.0090592Z             zk = get_fake_zk(node1.name, timeout=30.0)
2024-04-09T16:31:55.0091216Z             do_some_action(zk, create_cnt=10)
2024-04-09T16:31:55.0091721Z     
2024-04-09T16:31:55.0092456Z             data = node1.send_4lw_cmd(cmd='crst')
2024-04-09T16:31:55.0092891Z     
2024-04-09T16:31:55.0093332Z             print("crst output -------------------------------------")
2024-04-09T16:31:55.0093987Z             print(data)
2024-04-09T16:31:55.0094401Z     
2024-04-09T16:31:55.0094761Z             data = node1.send_4lw_cmd(cmd='cons')
2024-04-09T16:31:55.0095299Z             print("cons output -------------------------------------")
2024-04-09T16:31:55.0095942Z             print(data)
2024-04-09T16:31:55.0096323Z     
2024-04-09T16:31:55.0096792Z             # 2 connections, 1 for 'cons' command, 1 for zk
2024-04-09T16:31:55.0097510Z             cons = [n for n in data.split('\n') if len(n) > 0]
2024-04-09T16:31:55.0098143Z >           assert len(cons) == 2
2024-04-09T16:31:55.0098565Z E           AssertionError: assert 3 == 2
2024-04-09T16:31:55.0100347Z E            +  where 3 = len([' 172.23.0.1:54118(recved=0,sent=0)', ' 172.23.0.1:54108(recved=0,sent=0)', ' 172.23.0.1:54102(recved=0,sent=0,sid=0x2a,lop=NA,est=1712679539836,to=30000,lzxid=0xffffffffffffffff,lresp=0,llat=0,minlat=0,avglat=0,maxlat=0)'])
2024-04-09T16:31:55.0101821Z 
2024-04-09T16:31:55.0102036Z test_four_word_command/test.py:445: AssertionError

Connections:
```
ons output -------------------------------------
2024-04-09T16:31:55.0112721Z  172.23.0.1:54118(recved=0,sent=0)
2024-04-09T16:31:55.0113290Z  172.23.0.1:54108(recved=0,sent=0)
2024-04-09T16:31:55.0114522Z  172.23.0.1:54102(recved=0,sent=0,sid=0x2a,lop=NA,est=1712679539836,to=30000,lzxid=0xffffffffffffffff,lresp=0,llat=0,minlat=0,avglat=0,maxlat=0)
```
### How to reproduce 

https://productionresultssa2.blob.core.windows.net/actions-results/b29bfb4b-0061-43a1-beae-c842c33db215/workflow-job-run-8d1a6b72-f544-5292-134b-07751b9ec7d7/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-04-10T02%3A10%3A35Z&sig=IUyahY9zHMkLle8FQ%2BFbDwsF95%2BOAICMNBbSYwu%2FMo8%3D&sp=r&spr=https&sr=b&st=2024-04-10T02%3A00%3A30Z&sv=2021-12-02

--- a/tests/integration/test_four_word_command/test.py
+++ b/tests/integration/test_four_word_command/test.py
@@ -441,8 +441,9 @@ def test_cmd_crst(started_cluster):
         print(data)
 
         # 2 connections, 1 for 'cons' command, 1 for zk
+        # but if there is reconnection len(cons) may large than 2
         cons = [n for n in data.split('\n') if len(n) > 0]
-        assert len(cons) == 2
+        assert len(cons) >= 2
 
         zk_con = None
         if 'sid' in cons[0]:

--- a/tests/integration/test_four_word_command/test.py
+++ b/tests/integration/test_four_word_command/test.py
@@ -441,7 +441,7 @@ def test_cmd_crst(started_cluster):
         print(data)
 
         # 2 connections, 1 for 'cons' command, 1 for zk
-        # but if there is reconnection len(cons) may large than 2
+        # but len(cons) may be large than 2, if the connection of `crst` is not destroied
         cons = [n for n in data.split('\n') if len(n) > 0]
         assert len(cons) >= 2
 

